src/main/java/baritone/pathing/precompute/PrecomputedData.java

```diff
@@ -22,46 +22,47 @@
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.state.BlockState;
 
-import static baritone.pathing.precompute.Ternary.MAYBE;
-import static baritone.pathing.precompute.Ternary.YES;
-
 public class PrecomputedData {
 
-    private final int[] data = new int[Block.BLOCK_STATE_REGISTRY.size()];
+    private final byte[] data = new byte[Block.BLOCK_STATE_REGISTRY.size()];
+
+    /**
+     * byte layout
+     *
+     *          0              1             2              3              4              5              6             7
```

Comment:
This is big endian 👎

Reply:
(I think it would be more clear, relative to the subsequent lines, if "0" meant the least significant bit, because you could see that "1 << 0" corresponds with bit 0, "1 << 1" corresponds with bit 1, etc) (in other words, little endian ordering where "0" is the LSB not the MSB)